tests/VoTest/DateRangeTest.php

```diff
@@ -3,6 +3,7 @@
 namespace VoTest;
 
 use DateTime;
+use DateTimeImmutable;
```

Comment:
Mutable dates still need to be tested as it is the VO's job to make them immutable.